IOSSecuritySuite/FishHookChecker.swift

```diff
@@ -103,18 +104,18 @@ internal class FishHookChecker {
         let linkeditCmdName = SEG_LINKEDIT.data(using: String.Encoding.utf8)!.map({ $0 })
         var linkeditCmd: UnsafeMutablePointer<segment_command_64>!
         var dyldInfoCmd: UnsafeMutablePointer<dyld_info_command>!
-        
+
         // Text cmd
         let textCmdName = SEG_TEXT.data(using: String.Encoding.utf8)!.map({ Int8($0) })
```

Comment:
and also here, `guard let linkeditCmdName = SEG_LINKEDIT.data(using: String.Encoding.utf8)?.map({ $0 }),
                                     let textCmdName = SEG_TEXT.data(using: String.Encoding.utf8)?.map({ Int8($0) })
                  else { return }`